assets/scripts/index.js

```diff
@@ -0,0 +1,126 @@
+$(document).ready(function(){
+
+  document.querySelector(".js-nav-toggle").addEventListener("click",toggleMenuFunction);
+  document.querySelector(".js-top").addEventListener("click",toTop);
+  window.onscroll = function() {scrollFunction()};
```

Comment:
@edwartexe if you aren't going to send extra info to the function, you can write this as follows
```suggestion
  window.onscroll = scrollFunction;
```